### DISCUSSION TITLE
blog/goodbye-katacoda/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/rootsongjc.github.io#283] [Announcements]
# blog/goodbye-katacoda/

O'Reilly 宣布将于 6 月 15 日关闭在线学习网站 KataCoda。

https://jimmysong.io/blog/goodbye-katacoda/

**Comment by rootsongjc** [comment]
对KataCoda的关闭还是挺惋惜的，之前还在上面创建过几个课程，用来同步镜像，R.I.P 2016 - 2022

**Comment by zhouyijiaren** [comment]
No
